vpdq/python/vpdq.pyx

```diff
@@ -127,7 +136,7 @@ def computeHash(
         downsample_height = vid.get(cv2.CAP_PROP_FRAME_HEIGHT)
 
     rt = hashVideoFile(
-        input_video_filename.encode("utf-8"),
+        str(input_video_filename).encode("utf-8"),
```

Comment:
What does this encode do?

Reply:
The C++ function takes the encoded str.

Reply:
By encoded string, do you mean `bytes` / c-style?

Reply:
Yes